src/FondOfSpryker/Zed/ProductListsRestApi/Persistence/ProductListsRestApiRepository.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace FondOfSpryker\Zed\ProductListsRestApi\Persistence;
+
+use Generated\Shared\Transfer\ProductListTransfer;
+use Spryker\Zed\Kernel\Persistence\AbstractRepository;
+
+/**
+ * @method \FondOfSpryker\Zed\ProductListsRestApi\Persistence\ProductListsRestApiPersistenceFactory getFactory()
+ */
+class ProductListsRestApiRepository extends AbstractRepository implements ProductListsRestApiRepositoryInterface
+{
+    /**
+     * @param string $productListUuid
+     *
+     * @throws
```

Comment:
Can you specify the Exception: example  * @throws \Exception

Reply:
the coding style from spryker removes specific exceptions. @SpiderDan98 remove the throws tag.